Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor

```diff
@@ -127,14 +127,14 @@
                         {
                             @foreach ( var column in ParentDataGrid.GetColumns().Where( x => x.IsRegularColumn ) )
                             {
-                                <DropdownItem ShowCheckbox Checked=@column.Displaying CheckedChanged="@(value => ColumnDisplayingChanged( column, value ))">@column.Caption</DropdownItem>
+                                <DropdownItem ShowCheckbox Checked=@column.Displaying CheckedChanged="@(value => ColumnDisplayingChanged( column, value, column.DisplayOrder ))">@column.Caption</DropdownItem>

```

Comment:
This display order passing has no value. It will hit the `Column.SetDisplaying` where it will met the same displayOrder value.